orthopairs/src/main/resources/config.properties

```diff
@@ -1,4 +1,4 @@
-releaseNumber=
+releaseNumber=90
```

Comment:
Just wondering if releaseNumber should be set in the checked-in file and if it should be set to 90?

Reply:
No, it shouldn't be there